src/com/company/Main.java

```diff
@@ -12,14 +12,14 @@ public static void main(String[] args){
 
         String txtfile = args[0];
```

Comment:
Please check what will happen if args[0] does not contain anything. For example if user did not pass any argument what happens to the program?

Reply:
it display the file not exists message

Reply:
i have changed the code with dispalying the message if user does not passes any one output